src/index.css

```
@@ -72,19 +88,25 @@ body {
 }
 
 section {
-  width: 100dvw;
+  width: 100%;
 }
 
 @theme {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)